handbook/engineering/distribution/goals.md

```diff
@@ -20,12 +20,12 @@ Our existing deployment pipelines to our Sourcegraph instances (such as Sourcegr
   - TBD: We can trigger rollbacks and deployments via a `/` command in Slack.
 - **Milestones**: TBD
 
-### Any engineer at Sourcegraph can create a release for all of our supported deployment types by running a single command
+### Any engineer at Sourcegraph can create a release for all of our supported deployment types by running a single action
 
 Creating a new release for our deployments is currently a semi-automated process, which requires several manual steps and synchronizing our versioned artifacts (Sourcegraph, Kubernetes manifests, docker-compose manifests, etc). We want to enable any engineer to perform a release as often as needed, to enable this we want to make releasing Sourcegraph a simple, automated process.
 
 - **Owner**: Dave, Robert
-- **Status**: In progress. Estimated completion by end of 2020.
+- **Status**: In progress. Estimated completion by end of FY21-Q1.
```

Comment:
I updated this based on your earlier comment in https://github.com/sourcegraph/about/pull/1754#discussion_r503560574 @pecigonzalo